README.md

```diff
@@ -100,12 +100,15 @@ data:
   x.json: |
     {"a": [1,2,3,4]}
 ```
+Note: "x.json" may be any key.
 
 You could refer to the data inside your policies as follows:
 
 ```ruby
-data.opa["hello-data"]["x.json"].a[0]  # evaluates to 1
+import data.opa  # you may mock this in tests: `with data.opa as mock_object`
+opa["hello-data"]["x.json"].a[0]  # evaluates to 1
```

Comment:
You don't **have** to import it, so that it can be mocked. You could keep `data.opa["hello-data"]["x.json"].a[0]`, and still be able to override it using a `with data.opa as ...` expression, couldn't you?

Reply:
Aggreed